admin/governance.md

```diff
@@ -0,0 +1,60 @@
+# Repository governance
+
+This file contains notes on our tools and processes for working in this repo. It's meant as notes for the maintainers, but may be informative for other contributors as well.
+
+## Workflows installed
+
+We have the following workflows that run on most PRs:
+
+- **OpenPublishing.Build** *required*: This builds published docs site using the changes from the PR. It's how we verify changes before publishing. We can see any changes in a staging site before publishing.
+- **Snippets 5000 / snippets-build** *required*: This workflow scans the PR for any changes to source code. If any source code files are changed, it ensures the changes are part of a project, and then builds that project. We require all code snippets to be included in a project that produces a clean build on the current version of .NET (unless a config file pins that project to a known older version.)
+- **MSDocs build verifier / MSDocs build verifier**  *required*: This action verifies links and other docfx syntax for our build.
+- **OPS status checker / Look for build warnings** *required*: This action fails when the OPS build has warnings. That prevents auto-merge from merging PRs with warnings.
+- **license/cla**: This ensures that any contributor making large scale changes to our docs has signed the Contributor License Agreement (CLA).
+- **.github/workflows/live-protection.yml**: This action ensures that the only PRs that target the `live` branch use the `main` branch as the source. All work should take place against the `main` branch. Nothing should be merged to `live` that hasn't been published on the staging site in the `main` branch.
+- **Markdownlint / lint**: We like our markdown clean and spiffy. Please keep it that way.
+- **dependabot auto-approve and auto-merge**: We have dependabot installed to open PRs when referenced NuGet packages are updated. If the build is clean for those updates, those PRs are automatically merged by this action.
+
+## Branch protection rules
+
+We've also configured a variety of branch protection rules in place on the `main` branch:
+
+- All changes to the `main` branch must be through a PR (no direct push privileges to `main`)
+- All PRs must have at least one approval from a maintainer.
+- All required checks must pass.
+- All open conversations must be resolved.
+- Protection rules apply to everyone, including administrators.
+
+> On rare occasions, we may override the required checks. This should be rare, and only after exhausting other options.
+>
+> If you do need to override the required settings, turn off the "Do not allow bypassing the above settings*. Then, refresh the PR, merge it, and the reapply the "Do not allow bypassing" setting. This is only available to administrators.
```

Comment:
Should we make a note that if you change any branch protection rules, you must alert the .NET Content team via email? Maybe we don't want to put the actual email address here since it's public.